onair/src/ai_components/ai_plugin_abstract/ai_plugin.py

```diff
@@ -1,38 +1,63 @@
 # GSC-19165-1, "The On-Board Artificial Intelligence Research (OnAIR) Platform"
 #
-# Copyright © 2023 United States Government as represented by the Administrator of
-# the National Aeronautics and Space Administration. No copyright is claimed in the
-# United States under Title 17, U.S. Code. All Other Rights Reserved.
+# Copyright © 2023 United States Government as represented by the Administrator
+# of the National Aeronautics and Space Administration. No copyright is claimed
+# in the United States under Title 17, U.S. Code. All Other Rights Reserved.
 #
 # Licensed under the NASA Open Source Agreement version 1.3
 # See "NOSA GSC-19165-1 OnAIR.pdf"
 
+
 from abc import ABC, abstractmethod
-"""This object serves as a proxy for all plug-ins.
-   Therefore, the AIPlugIn object is meant to induce
-   standards and structures of compliance for user-created
-   and/or imported plug-ins/libraries
-"""
+
+
 class AIPlugIn(ABC):
-    def __init__(self, _name, _headers):
-        """
-        Superclass for data driven components: VAE, PPO, etc. Allows for easier modularity.
+    """This serves as a base for all plugins.
+
+    This is the superclass for data driven components: VAE, PPO, etc. It is
```

Comment:
We should probably update the examples to ones we have... which is none (the Kalman plugin doesn't count, right?)... so I don't know.

We could at least get rid of the acronyms.

Reply:
I'd be in favor of just removing that sentence for now.

Reply:
Done.